### ISSUE TITLE
Syntax error when both static class and data attribute are used

### ISSUE THREAD

**Issue by eagletmt** [k0kubun/hamlit#17]
```haml
- h = { bar: 'baz' }
%span.foo{data: h}
```

```
% bundle exec hamlit compile in.haml
_buf = []; h = { bar: 'baz' }; 
; _buf << ("<span".freeze); _buf << (::Hamlit::Attribute.build("'", data: h, {:class=>"foo"})); _buf << ("></span>\n".freeze); 
; _buf = _buf.join
```

It's caused by https://github.com/k0kubun/hamlit/commit/7d3056b89aa7fc6cedd9c534dca24f6e0109ca02 .

**Comment by k0kubun** [comment]
Thank you for your testing.
Fixed and released as v0.5.3.

**Comment by eagletmt** [comment]
:+1:
